### ISSUE TITLE
GFF3toolkit (installation request)

### ISSUE THREAD

**Issue by mhalfon** [ubccr/software-layer#321]
I'd like to request installation of GFF3toolkit. It looks like there is an easybuild recipe here:
https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/g/GFF3-toolkit/GFF3-toolkit-2.1.0-foss-2022a.eb

Source as well as example/test files can be found here:
https://github.com/NAL-i5K/GFF3toolkit/tree/master?tab=readme-ov-file

Thank you!

**Comment by mhalfon** [comment]
Thanks Tony!

Marc S. Halfon, PhD
Professor of Biochemistry
University at Buffalo-State University of New York

Sent from my iPhone

On Jun 6, 2024, at 2:05 PM, Tony Kew ***@***.***> wrote:

﻿

GFF3-toolkit has been built, and is going through internal testing.
I will update this request once GFF3-toolkit has been published.

**Comment by tonykew** [comment]
GFF3-toolkit version 2.1.0 has been published to the ccrsoft/2023.01 software release

```
login2$ module spider gff3-toolkit

----------------------------------------------------------------------------
  gff3-toolkit:
----------------------------------------------------------------------------
    Description:
      Python programs for processing GFF3 files

     Versions:
        gff3-toolkit/2.1.0

[...]
login2$ module spider gff3-toolkit/2.1.0

----------------------------------------------------------------------------
  gff3-toolkit: gff3-toolkit/2.1.0
----------------------------------------------------------------------------
    Description:
      Python programs for processing GFF3 files


    You will need to load all module(s) on any one of the lines below before the "gff3-toolkit/2.1.0" module is available to load.

      gcc/11.2.0  openmpi/4.1.1
[...]
login2$ module load gcc/11.2.0 openmpi/4.1.1 gff3-toolkit/2.1.0
login2$ ls ${EBROOTGFF3MINTOOLKIT}/bin
gff3_fix  gff3_merge  gff3_QC  gff3_sort  gff3_to_fasta
login2$ 
```

e.g.

```
login2$ gff3_QC --help
usage: gff3_QC [-h] [-g GFF] [-f FASTA] [-noncg] [-i] [-n ALLOWED_NUM_OF_N]
               [-t [CHECK_N_FEATURE_TYPES ...]] [-o OUTPUT] [-s STATISTIC]
               [-v]
[...]
login2$ 
```
